selftests/unit/test_test.py

```diff
@@ -98,20 +98,20 @@ def check(uid, name, variant, exp_logdir):
 
     def test_data_dir(self):
         """
-        Tests that a valid datadir exists following the test filename
+        Checks `get_data()` won't report fs-unfriendly data dir name
```

Comment:
well this checks that it reports fs-friendly data dir so this is not incorrect, but doesn't really describe this test (it fits the next test better). But let's get this in as it's still better than previously existing docstring...